packages/node_modules/@ciscospark/widget-meet/src/messages.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * WidgetMeet Messages
+ *
+ * This contains all the text for the FeaturePage component.
+ */
+import {defineMessages} from 'react-intl';
+
+export default defineMessages({
```

Comment:
doesn't defining the defaults this far from the element defeat the purpose of react-intl? I thought the goal was more to define defaults at site of use, that way anyone looking at the code for the first time would understand the intent.